.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: Lint
+        run: |
+          yarn
+          yarn lint
+      - name: '[TEMPORARY] Modify Package Visibility'
+        env:
+          NPM_TOKEN: ${{secrets.LIFEOMIC_NPM_TOKEN}}
+        run: |
+          echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > .npmrc
+          npm access public @lifeomic/typescript-config
```

Comment:
Will remove this in a subsequent PR, per https://lifeomic.slack.com/archives/C0137E2L8C9/p1634664616057000?thread_ts=1634583519.030900&cid=C0137E2L8C9